dianna/methods/shap.py

```diff
@@ -1,21 +1,223 @@
 import numpy as np
+import shap
+import onnx
+from skimage.segmentation import slic
+from onnx_tf.backend import prepare  # onnx to tf model converter&runner
+import warnings
 
 
-class SHAP:
+class KernelSHAP:
     """
-    SHAP implementation based on captum https://github.com/pytorch/captum
+    Kernel SHAP implementation based on shap https://github.com/slundberg/shap
     """
-    def __init__(self, shap_arg_1=0, shap_arg_2=0):
+
+    def __init__(self):
+        """KernelSHAP initializer.
+
         """
-        Example constructor for testing.
+
+    def _segment_image(
```

Comment:
This function doesn't use `self`, so it can be moved outside of the class as a generic helper function.